Description: Increase FFMPEG_INPUT_BUFFER_PADDING_SIZE to 32
 This is needed to be able to build with FFmpeg 2.3.
Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Last-Update: 2014-07-20
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=758359
---
 libmpdemux/demuxer.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- mplayer2.orig/libmpdemux/demuxer.h
+++ mplayer2/libmpdemux/demuxer.h
@@ -121,7 +121,7 @@ enum timestamp_type {
 #define SEEK_BACKWARD (1 << 3)
 
 // demux_lavf can pass lavf buffers using FF_INPUT_BUFFER_PADDING_SIZE instead
-#define MP_INPUT_BUFFER_PADDING_SIZE 16
+#define MP_INPUT_BUFFER_PADDING_SIZE 32
 
 typedef struct demux_stream {
     int buffer_pos;        // current buffer position
